tests/mock/AnnotatorPageTest.php

```diff
@@ -1,19 +1,28 @@
-<?php
-
-
-class AnnotatorPageTest extends Page implements TestOnly
-{
-    private static $db = array(
-        'SubTitle'    => 'Varchar(255)'
-    );
-}
-
-class AnnotatorPageTest_Controller extends Page_Controller implements TestOnly
-{
-
-}
-
-class AnnotatorPageTest_Extension extends Extension implements TestOnly
-{
-
-}
+<?php
+
+namespace Axyr\IDEAnnotator\Tests;
+
+require_once('/mysite/code/Page.php');
+require_once('/mysite/code/PageController.php');

```

Comment:
We shouldn't need to do this. If this is to fix problems in CI then you should switch to using `silverstripe/installer` instead of `silverstirpe/recipe-core` or `*/recipe-cms` since installer will run the mysite bootstrapping for you

Reply:
I have no idea why it was actually needed, it confused the hell out of me tbh. There is a lot of unfinished stuff in here for SS4, so a lot of temporary fixes before the final stuff.